cypress/integration/sidebar-navigation/index.spec.ts

```diff
@@ -56,6 +56,19 @@ describe('Sidebar navigation', () => {
   });
 });
 
+describe('Sidebar opens correct section on first load', () => {
+  it('opens the sidebar category and selects the page correctly on first load', () => {
```

Comment:
answer: it will happen on first load on any deep link, which you can see here: https://paste.twilio.design/primitives/box

The nav will work as you navigate through the website, but I've consistently observed deep links / refreshes dropping the nav context. Test seems good from my perspective.